dolphinscheduler-ui/src/views/projects/workflow/definition/components/start-modal.tsx

```diff
@@ -144,6 +147,11 @@ export default defineComponent({
       }
     ]
 
+    const showTaskDependType = computed(() =>
+      (route.name === 'workflow-definition-detail' ||
+        route.name === 'workflow-instance-detail')
+    )
+
```

Comment:
Is this only displayed on page workflow definition detail? because the start action is not supported on workflow instance detail page.

Reply:
> Is this only displayed on page workflow definition detail? because the start action is not supported on workflow instance detail page.

Ok, i'll make some changes